<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors Difference</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Open1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Open2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Image1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>242</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Image2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="Loading">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ResultImage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Created by Robert Liberski&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Save"/>
    <addaction name="SaveAs"/>
    <addaction name="Close"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
